docs/application-connector/10-02-application-gateway-troubleshooting.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Error when calling a registered Service
+type: Troubleshooting
+---
+
+If you call a registered service and receive an error:
+
+- Verify that the call reached Application Gateway.  
+  To do that, fetch logs from Application Gateway Pod:
+  ```
+  kubectl -n kyma-integration logs -l app={APP_NAME}-application-gateway -c {APP_NAME}-application-gateway
+  ```
+  If the request reaches the Pod, it is logged by Application Gateway.
+  
+  If the call is not in the logs, check if [Access Service](components/application-connector/#architecture-application-connector-components-access-service) for the Service you are trying to call exists.
+  ```
+  kubectl -n kyma-integration get svc app-{APP_NAME}-{SERVICE_ID}
+  ```
+  If Access Service does not exist, run this command to deregister the Service:
+
+  <div tabs name="deregistration">
+    <details>
+    <summary>
+    With trusted certificate
+    </summary>
+
+    ```
+    curl -X DELETE https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/metadata/services/{SERVICE_ID} --cert {CERTIFICATE_FILE} --key {KEY_FILE}
+    ```
+    </details>
+    <details>
+    <summary>
+    Without trusted certificate
+    </summary>
+
+    ```
+    curl -X DELETE https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/metadata/services/{SERVICE_ID} --cert {CERTIFICATE_FILE} --key {KEY_FILE} -k
+    ```
+    </details>
+  </div>
+
+  Then register the Service.  
+  To register a Service, see [this tutorial](components/application-connector/#tutorials-register-a-service-register-a-service) again.
```

Comment:
Don't use "again". 
The person who gets this error might not be the same one who registered the service.